packages/strapi-design-system/src/SubNav/SubNav.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Grid } from '../Grid';
+
+const SubNavWrapper = styled(Grid)`
+  width: ${232 / 16}rem;
+  background: ${({ theme }) => theme.colors.neutral100};
+  height: 100%;
+  position: relative;
+  overflow-y: auto;
+`;
+
+export const SubNav = ({ condensed, ...props }) => {
+  return <SubNavWrapper as="nav" rows="auto auto 1fr auto auto" condensed={condensed} {...props} />;
```

Comment:
do you think we should add an explicit API for the label ? it can be added directly in the implementation `<SubNav aria-label="Builder sub navigation`

Reply:
It depends: if it can be used by external people, I think we should enforce it. If it's for us only, I think we can rely on aria-label.

WYT?

Reply:
You are right, I will add an `ariaLabel` required prop to ensure that the plugin sub navigations are labelled.